README.md

```diff
@@ -21,33 +21,33 @@ To get a feel for what an UberFx service looks like, see our
 
 ## Service Model
 
-A service is a container for a set of **modules**, controlling their lifecycle.
-Service can have any number of modules that are responsible for a specific type
+A service is a container for a set of **modules** and controls their lifecycle.
+A service can have any number of modules, each responsible for a specific type
 of functionality, such as a Kafka message ingestion, exposing an HTTP server, or
 a set of RPC service endpoints.
 
 The core service is responsible for loading basic configuration and starting and
-stopping a set of these modules.  Each module gets a reference to the service to
-access standard values such as the Service name or basic configuration.
+stopping a set of these modules. Each module gets a reference to the service to
+access standard values such as the service name or basic configuration.
 
 [Read more about the service model](service/README.md)
 
-## Core packages
+## Core Packages
 
-The top-level packages (except for `service`) contain the nuts and bolts useful
-to have in a fully-fledged service, but are not specific to an instance of a
-service or even the idea of a service.
+The top-level packages contain the nuts and bolts useful for a fully-fledged
+service. That said, none (except for `service`) requires an instance or even
+the idea of a service. You can use the packages independently their logic.
```

Comment:
Value sounds like a number, but good catch. Is "for their functionality" too clunky? Or just end the sentence after "independently"?